onecore_maintenance_extension/models/maintenance.py

```diff
@@ -33,60 +33,60 @@ class OneCoreMaintenanceRequest(models.Model):
 
     #    RENTAL PROPERTY  ---------------------------------------------------------------------------------------------------------------------
 
-    rental_property_id = fields.Many2one('maintenance.rental.property', store=True, string='Rental Property Id')
+    rental_property_id = fields.Many2one('maintenance.rental.property', store=True, string='Hyresobjekt ID')
```

Comment:
Tror Hyresobjekt räcker? 🙂 

```suggestion
    rental_property_id = fields.Many2one('maintenance.rental.property', store=True, string='Hyresobjekt')
```